AppInspector/rules/default/tools/pipeline.json

```diff
@@ -0,0 +1,325 @@
+        "scopes": [ "code", "comment" ],
+        "modifiers": [ "i" ],
+        "confidence": "high",
+        "_comment": ""
+      }
+    ]
+  },
+  {
+    "name": "Pipeline Tool (AntiMalware)",
+    "id": "AI053500",
+    "description": "Authentication using Microsoft Identity Platform (ADAL/MSAL) or similar",
+    "tags": [
+      "Metadata.Pipeline.Tools.AntiMalware"
+    ],
+    "severity": "critical",
+    "patterns": [
+      {
+        "pattern": "AntiMalware",
+        "type": "string",
+        "scopes": [ "code", "comment" ],
+        "modifiers": [ "i" ],
+        "confidence": "high",
+        "_comment": ""
+      }
+    ]
+  },
+  {
+    "name": "Pipeline Tool (CodeSigning)",
+    "id": "AI053600",
+    "description": "Authentication using Microsoft Identity Platform (ADAL/MSAL) or similar",
+    "tags": [
+      "Metadata.Pipeline.Tools.CodeSigning"
+    ],
+    "severity": "critical",
+    "patterns": [
+      {
+        "pattern": "EsrpCodeSigning",
+        "type": "string",
+        "scopes": [ "code", "comment" ],
+        "modifiers": [ "i" ],
+        "confidence": "high",
+        "_comment": ""
+      }
+    ]
+  },
+  {
+    "name": "Pipeline Tool (RoslynAnalyzers)",
+    "id": "AI053700",
+    "description": "Roslyn",
+    "tags": [
+      "Metadata.Pipeline.Tools.RoslynAnalyzers"
+    ],
+    "severity": "critical",
+    "patterns": [
+      {
+        "pattern": "RoslynAnalyzers@2",
+        "type": "string",
+        "scopes": [ "code", "comment" ],
+        "modifiers": [ "i" ],
+        "confidence": "high",
+        "_comment": ""
+      }
+    ]
+  },
+  {
+    "name": "Pipeline Tool (TSLint)",
+    "id": "AI053800",
+    "description": "SpotBugs",
+    "tags": [
+      "Metadata.Pipeline.Tools.TSLint"
+    ],
+    "severity": "critical",
+    "patterns": [
+      {
+        "pattern": "TSLint@1",
+        "type": "string",
+        "scopes": [ "code", "comment" ],
+        "modifiers": [ "i" ],
+        "confidence": "high",
+        "_comment": ""
+      }
+    ]
+  },
+  {
+    "name": "Pipeline Tool (CredScan)",
+    "id": "AI053900",
+    "description": "SpotBugs",
+    "tags": [
+      "Metadata.Pipeline.Tools.VulnerabilityAssessment"
+    ],
+    "severity": "critical",
+    "patterns": [
+      {
+        "pattern": "VulnerabilityAssessment@0",
+        "type": "string",
+        "scopes": [ "code", "comment" ],
+        "modifiers": [ "i" ],
+        "confidence": "high",
+        "_comment": ""
+      }
+    ]
+  },
+  {
+    "name": "Pipeline Tool (CredScan)",
+    "id": "AI054000",
+    "description": "SpotBugs",
+    "tags": [
+      "Metadata.Pipeline.Tools.SpotBugs"
+    ],
+    "severity": "critical",
+    "patterns": [
+      {
+        "pattern": "SpotBugs@1",
+        "type": "string",
+        "scopes": [ "code", "comment" ],
+        "modifiers": [ "i" ],
+        "confidence": "high",
+        "_comment": ""
+      }
+    ]
+  },
+  {
+    "name": "Pipeline Tool (CredScan)",
```

Comment:
Mismatch -- I think this whole file needs another run through.

Reply:
Looks like it :(